### PR TITLE
Managing multiple organisations

### PR DESCRIPTION
**Pull request by MartinBentley**
### Context

This is a breaking change, the matching UI PR is https://github.com/DFE-Digital/manage-courses-ui/pull/72

### CODE REVIEW

File: src/ManageCourses.Api/Data/DataService.cs [hunk 1]
```diff
@@ -247,6 +249,16 @@ public OrganisationCourses GetCoursesForUser(string email)
 
             return returnCourses;
         }
+        /// <summary>
+        /// Gets a list of organisation the user in linked to
+        /// </summary>
+        /// <param name="email"></param>
+        /// <returns></returns>
```
Comment (defong) [inline src/ManageCourses.Api/Data/DataService.cs]: If you doing xml doc can you fill it in properly please

File: src/ManageCourses.Api/Data/DataService.cs [hunk 2]
```diff
@@ -247,6 +249,11 @@ public OrganisationCourses GetCoursesForUser(string email)
 
             return returnCourses;
         }
+        public IEnumerable<UserOrganisation> GetOrganisationsForUser(string email)
```
Comment (defong) [inline src/ManageCourses.Api/Data/DataService.cs]: Is there a reason for this to be public?

**Comment by defong** [comment]
Break out the test data in teest class to stand on its own.
If the class test is still too big
Break the test method in standalone class as well

**Comment by timabell** [comment]
I think this might be returning the wrong orgIds, see comment on UI PR

**Pull request merged**